cluster-provision/k8s/1.18/provision.sh

```diff
@@ -87,7 +87,7 @@ cat << EOF > /etc/docker/daemon.json
   "log-driver": "json-file",
   "exec-opts": ["native.cgroupdriver=systemd"],
   "ipv6": true,
-  "fixed-cidr-v6": "2001:db8:1::/64",
+  "fixed-cidr-v6": "2001:db9:1::/64",
```

Comment:
if we have 2 nodes. do we need to change it for the 2nd node ?
(after cluster-up of course)

Reply:
It should be "db9" on all the nodes. Have I missed a place to change?

Reply:
you didn't miss, just in case it was need to be different, then other files would need to be updated.